config/locales/views/admin/fr.yml

```diff
@@ -10,6 +10,14 @@ fr:
           no_flags: Article has no flags.
           no_quality_reactions: Article has no quality reactions by trusted users.
           title: Moderator actions
+      comments:
+        flags:
+          score: Score affecté par un drapeau particulier
+        priviliged_actions:
+          description: Toutes les actions du modérateur affectent le score. Le score global est une combinaison des actions des modérateurs et des réactions du public.
+          no_flags: Le commentaire n'a pas de drapeaux.
+          no_quality_reactions: Le commentaire n'a pas de réactions de qualité par des utilisateurs de confiance.
+          title: Actions du modérateur        
```

Comment:
Just noting that in French `drapeau` is strictly flag as in the cloth you put on a pole; for the sense we are using here (marked/reported by someone) we would use `indicateur`/`signe` (noun) and `signaler` (verb)

I could go over this in a separate PR because I don't want to hold this one up on back-on-forth about the best wording. Also that way I can check/correct other translations as well

Reply:
I think we need some form of guidelines here. Who does all these translations? I did few translations in my earlier PRs but found out that plurals work differently in french vs english and therefore using google translate is not the most correct solution. 
No worries. Will take this question to slack threads. Thanks.

Reply:
We do not have a process in place to review translations unfortunately as it hasn't been a priority and we didn't have in-house expertise to review translations. That said, if we have fluency on the team, we can have at least a better review process for French.